tests/unit_tests/test_elasticsearchstore.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import mock
+import mlflow
+from mlflow.entities.lifecycle_stage import LifecycleStage
+import elasticsearch_dsl
+
+import mlflow_elasticsearchstore
+from mlflow_elasticsearchstore.elasticsearch_store import ElasticsearchStore
+from mlflow_elasticsearchstore.models import ElasticExperiment, ElasticRun, \
+    ElasticMetric, ElasticParam, ElasticTag
+
+
+@mock.patch('elasticsearch_dsl.Document')
+def test_create_experiment(document_mock):
+    store = ElasticsearchStore("user", "password", "host", "port")
+    store.create_experiment("name", "artifact_location")
```

Comment:
Please check the return value of this call with an assert at the end of the test